test/passing/tests/comments_around_disabled.ml

```diff
@@ -1,7 +1,7 @@
-(* cmts *)
+(* cmts before *)
 
 [@@@ocamlformat "disable"]
 let () = ()
```

Comment:
It would be better if this was something that is obviously not formatted. With a line break in the middle for example.

Reply:
done